src/components/separators/HorizontalSeparator.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const HorizontalSeparator = ({spaced, short}) => {
+
+  const separatorClass = classNames('sg-horizontal-separator', {
+    'sg-horizontal-separator--spaced': spaced && !short,
+    'sg-horizontal-separator--short-spaced': spaced && short
```

Comment:
only short do nothing and can be confused, maybe TYPE:{normal, spaced, short-spaced}